lib/presentation/home/home_screen.dart

```diff
@@ -22,65 +23,94 @@ class _HomePageState extends State<HomePage> {
     showOnboarding();
   }
 
+  int index = 0;
+  Widget? pageToShow;
   @override
   Widget build(BuildContext context) {
-    final _advancedDrawerController = AdvancedDrawerController();
-
-    return MenuDrawer(
-      advancedDrawerController: _advancedDrawerController,
-      child: Scaffold(
-        appBar: AppBar(
-          elevation: 0,
-          backgroundColor: Colors.white,
-          leading: Padding(
-            padding: const EdgeInsets.only(left: 14.0),
-            child: IconButton(
-              icon: const Image(
-                  image: AssetImage('assets/images/icons/menu_icon.png')),
-              onPressed: () => _advancedDrawerController.showDrawer(),
-            ),
-          ),
-        ),
-        body: SafeArea(
-          child: Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 23.0),
-            child: Column(
-              children: [
-                Expanded(
-                  child: SingleChildScrollView(
-                    child: Column(
-                      children: [
-                        Row(
-                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                          children: [
-                            const Text(
-                              'Upcoming crowdactions',
-                              style: TextStyle(
-                                  fontWeight: FontWeight.w500, fontSize: 18.0),
-                            ),
-                            TextButton(
-                              onPressed: () => context.router
-                                  .push(const CrowdActionBrowseRoute()),
-                              child: const Text(
-                                'View all',
-                                style: TextStyle(
-                                    fontWeight: FontWeight.w500,
-                                    fontSize: 14.0,
-                                    color: kIrisColor),
-                              ),
-                            ),
-                          ],
-                        ),
-                        const CrowdActionCarousel(),
-                      ],
-                    ),
-                  ),
-                ),
-              ],
-            ),
-          ),
+    if (index == 0) {
+      pageToShow = const HomePageScreen();
+    } else if (index == 1) {
+      pageToShow = const DemoTabPage();
+    } else if (index == 2) {
+      pageToShow = const DemoTabPage();
+    }
+    return Scaffold(
+      appBar: AppBar(
+        elevation: 0,
+        backgroundColor: Colors.white,
+      ),
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 23.0),
+          child: pageToShow,
+          // Column(

```

Comment:
Let's just remove the commented code, we have it in earlier commits.

Reply:
Also delete the menu drawer if it's no longer being used anywhere. We still have the code in earlier commits.